services/management/rds.go

```diff
@@ -314,6 +314,17 @@ func (s *RDSService) AddRDS(ctx context.Context, req *managementpb.AddRDSRequest
 			}
 			res.Mysql = invService.(*inventorypb.MySQLService)
 
+			if isPushMode(req.MetricsMode) {
```

Comment:
id we always run agents on pmm-server we can always use pull mode here

Reply:
let's remove this if and always set metrics mode PULL

Reply:
and replace comment in pmm repo that we don't use this value and it's always pull https://github.com/percona/pmm/blob/PMM-2.0/api/managementpb/rds.proto#L128
@AlekSi what do you think?

Reply:
That API does not allow one to start rds_exporter on any Node other than PMM Server Node? If yes, then we should always use pull for both pull and auto in that particular case, and return error if caller requested push. We can't silently ignore and "fix" user's error.

But in general, rds_exporter can be started on any Node via Inventory API.

Reply:
1. yes, ok, let's return error and update comment in pmm repo.